make_changelog.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+import re
+import sys
+
+# Read existing changelog
+with open("Changelog.md") as f:
+    changelog_lines = list(s.rstrip() for s in f)
+
+# Determine new version
+this_version = sys.argv[1]
+today = datetime.date.today()
+today_str = f"{today.strftime('%Y')}-{today.strftime('%m')}-{today.strftime('%d')}"
+
+# Determine last version
+matches = re.finditer(r"^## \[(.*)\].*$", "\n".join(changelog_lines), re.MULTILINE)
```

Comment:
```suggestion
matches = re.finditer(r"^## \[v?(.*)\].*$", "\n".join(changelog_lines), re.MULTILINE)
```

I just tested this and noticed that the changelog sometimes includes the "v" and sometimes it doesn't (which causes multiple problems, such as incorrect GH urls or the `last_version == this_version` check not working). Stripping the v away, makes the script more robust.